tests/benchmarks/multicore/idle_with_pwm/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _multicore_idle_with_pwm_test:
+
+PWM in low power states test
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The test benchmarks the idle behavior of an application that runs on multiple cores and uses PWM driver to light up LED.
+
+Test scenario
+Repeat forever:
+- gradually light up LED (this lasts for ~1 second),
+- disable PWM (set period to 0); This will turn off LED,
+- sleep for 1 second.
+
+Requirements
+************
+
+The test supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk_nrf54h20_cpuapp
+
+Overview
+********
+
+The test demonstrates use of PWM with low power modes.
+
+Code stored in :file:`main.c` is compiled for Application and Radio cores.
+Application core uses pwm130 to generate PWM signal on LED2.
+Radio core uses pwm131 to generate PWM signal on GPIO Port 0 pin 7.
+
+There are three test configurations in :file:`testcase.yaml`.
+
+#. 'benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.s2ram'
+   This configuration uses KConfigs that enable entering low power modes. However, logging is disabled here.
+   Core sleeps for time sufficient to enter "suspend-to-ram" power state.
+#. 'benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.sleep'
+   This configuration uses KConfigs that enable entering low power modes. However, logging is disabled here.
+   Core sleeps for time sufficient to enter "suspend-to-idle" power state.
+#. 'benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.no_sleep'
+   This configuration can be used for debug purposes. It has enabled logging while Power Moding is disabled.
+
+Building and running
+********************
+
+.. |test path| replace:: :file:`tests/benchmarks/multicore/idle_with_pwm`
+
+.. include:: /includes/build_and_run_test.txt
+
+To build the test, use configuration setups from :file:`testcase.yaml` using the ``-T`` option.
+See the following examples:
+
+nRF54H20 DK
+  You can build the test for application and radio cores as follows:
+
+  .. code-block:: console
+
+     west build -p -b nrf54h20dk/nrf54h20/cpuapp -T benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.s2ram .
+
+Testing
+=======
+
+After programming the test to your development kit, complete the following steps to test it:
+
+#. Connect the PPK2 Power Profiler Kit or other current measurement device.
+#. Reset the kit.
+#. Observe the LED2 brightness (or use oscilloscope to collect PWM signal on Port 0 pin 7).
+#. When LED2 is OFF power profiler shall indicate low current consumption resulting from both cores being in low power mode.
```

Comment:
```suggestion
1. Connect the PPK2 Power Profiler Kit or other current measurement device.
#. Reset the kit.
#. Observe the **LED2** brightness (or use oscilloscope to collect the PWM signal on port 0, pin 7).
#. When the **LED2** is off, the power profiler indicates low current consumption resulting from both cores being in low power mode.
```

Reply:
I see that first line starts with ``1.``, while all other starts with ``#.``
Is this intentional?
**1.** Connect the PPK2 Power Profiler Kit...

Reply:
Yes, this is intentional. We start the first numbered item with 1. and the rest come just with hashes.